src/cards/ResourceCard/ResourceCard.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography, Button, Card, CardHeader, CardContent, CardActions, Tooltip, Fade } from '@material-ui/core';
+import { DefaultAvatar } from '../../misc';
+
+export const ResourceCard = (props) => {
+  const { id, name, description, action } = props;
```

Comment:
You can directely destucturing props like this :
port const ResourceCard = ({ id, name, description, action }) => {